src/v2/Apps/ViewingRoom/Components/ViewingRoomsLatestGrid.tsx

```diff
@@ -135,7 +140,18 @@ export const ViewingRoomsLatestGrid: React.FC<ViewingRoomsLatestGridProps> = pro
             const tag = getTagProps(status, distanceToOpen, distanceToClose)
 
             return (
-              <Link href={`/viewing-room/${slug}`} key={slug} noUnderline>
+              <Link
+                href={`/viewing-room/${slug}`}
+                key={slug}
+                noUnderline
+                id={
+                  // Calling loadMore loads 12 VRs at a time. We want to add a css id selector to the last VR in the list (list length - 12)
+                  // before loadMore was called and scroll to that element when the Show More button is pressed.
+                  viewingRoomsForLatestGrid.length - 12 === index
+                    ? "jump--viewingRoom"
```

Comment:
Rather than tacking this on to the link element, maybe add an empty element witht an id just above and only render it if this condition is met?